src/cmakeTaskProvider.ts

```diff
@@ -518,13 +518,13 @@ export class CustomBuildTaskTerminal implements vscode.Pseudoterminal, proc.Outp
                     return;
                 }
             }
-            const result: number | null | undefined = cmakeDriver ? await project?.runCTestCustomized(cmakeDriver, testPreset, this) : undefined;
-            if (result === undefined || result === null) {
+            const result: number | undefined = cmakeDriver ? await project?.runCTestCustomized(cmakeDriver, testPreset, this) : undefined;
+            if (result === undefined) {
                 this.writeEmitter.fire(localize('ctest.run.terminated', 'CTest run was terminated') + endOfLine);
                 this.closeEmitter.fire(-1);
             } else {
-                this.writeEmitter.fire(localize('ctest.finished.with.code', 'CTest finished with return code {0}', result) + endOfLine);
```

Comment:
Would it be useful to continue to log the exit code for diagnosing CTest issues?

Reply:
The original error is the exit code of ctest.exe. But now we invoke ctest for each test run, this exit code doesn't make sense anymore, and so this number returned just shows succeeded/failed status. But this status should be very obvious in the test explorer. And therefore, the exit code is removed here.